### DISCUSSION TITLE
My colorcheme is incorrect

### DISCUSSION THREAD

**Discussion by MykolaVaskevych** [arcticicestudio/nord#224] [Help & Support] [answered]

**Comment by arcticicestudio** [comment]
Hi @VaskMykola :wave:

Based on the color from the screenshot my first guess is that you're missing the actual color theme for your terminal.
Please check if there's a Nord port for your terminal available and install it if so, otherwise post the name of your terminal.

**Comment by svengreb** [comment]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.

---

@VaskMykola Would be nice if you could quickly check if this discussion can be closed since this is not related to this theme‘s code.

**Reply by MykolaVaskevych** [reply, marked as answer]
It’d have been a while. I’m not really using it anymore. Also, I am embarrassed about my previous message. If you don't mind, could you please delete/hide this discussion from others? 

Update:
I have deleted my previous message and closed this discussion.

Thank you
